orchagent/aclorch.cpp

```diff
@@ -1168,6 +1200,13 @@ bool AclTable::create()
         table_attrs.push_back(attr);
     }
 
+    if (type == ACL_TABLE_L3)
+    {
+        attr.id = SAI_ACL_TABLE_ATTR_FIELD_OUT_PORTS;
+        attr.value.booldata = true;
+        table_attrs.push_back(attr);
```

Comment:
I do not think OUT_PORTS is supported in other asic vendors, this code will break other platforms.

Reply:
OUT_PORTS may be supported by MCLAG, this rule may be supported on all ACL tables, but OUT_PORTS may be not supported by all vendors, so only support this on L3 table currently. What is the best way to fix this issue? Thinks.

Reply:
Yes , we will define new table dedicated for MCLAG. So it wouldn't affect others in case that mclag isn't enabled.

Reply:
Done